tests/unit/test_videos_core.py

```
@@ -0,0 +1,213 @@
+
+        find_large_videos(str(tmp_path), 0.00001, None, constants.SIZE_UNIT_GB)
+
+    def test_find_large_videos_handles_permission_errors(self, tmp_path: Path) -> None:
+        """Test find_large_videos handles permission errors gracefully."""
+        shutil.copyfile(FIXTURES_DIR / "large_2k.mp4", tmp_path / "large.mp4")
+
+        try:
+            (tmp_path / "large.mp4").chmod(0o000)
+            find_large_videos(str(tmp_path), 0.001, None, constants.SIZE_UNIT_MB, verbose=True)
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)